Libraries/Modal/__tests__/__snapshots__/Modal-test.js.snap

```diff
@@ -0,0 +1,79 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Modal /> should render as <Modal> when mocked 1`] = `
+<Modal
+  hardwareAccelerated={false}
+  visible={true}
+>
+  <View />
+</Modal>
+`;
+
+exports[`<Modal /> should render as <RCTModalHostView> when not mocked 1`] = `
+<RCTModalHostView
+  animationType="none"
+  hardwareAccelerated={false}
+  identifier={1}
+  onStartShouldSetResponder={[Function]}
+  presentationStyle="fullScreen"
+  style={
+    Object {
+      "position": "absolute",
+    }
+  }
+>
+  <View
+    style={
+      Array [
+        Object {
+          "left": 0,
+          "position": "absolute",
+          "top": 0,
+        },
+        Object {
+          "backgroundColor": "white",
+        },
+      ]
+    }
+  >
+    <View
+      pointerEvents="box-none"
+      style={
+        Object {
+          "flex": 1,
+        }
+      }
+    >
+      <View
+        collapsable={true}
+        pointerEvents="box-none"
+        style={
+          Object {
+            "flex": 1,
+          }
+        }
+      >
+        <View />
+      </View>
+    </View>
+  </View>
+</RCTModalHostView>
+`;
+
+exports[`<Modal /> should shallow render as <Component> when mocked 1`] = `
+<Component
```

Comment:
This is an example of displayName being broken. We’ll need to fix this. Can you either fix it in this PR or change the name of this test to be consistent with the other component tests so it is clear this isn’t correct?

Reply:
🤔 hmm modal is a mocked component. Are you sure this line isn't what's causing this? 

https://github.com/facebook/react-native/blob/master/jest/mockComponent.js#L20

Reply:
Yeah, that’s probably it but my understanding was the name line in the render function of that file is the thing that should be setting the component’s display name. Interestingly, that doesn’t seem to show up in this snapshot test at all.

Reply:
Sorry this took a bit. This goes back to my original PR, and this is related to what prompted all of this 😬 I tried explaining it but confused myself and deleted it all. I've been doing a bad job explaining it thus far so I think I need some time to put together a cohesive description.